### PR TITLE
Add breadcrumbs

### PR DESCRIPTION
**Pull request by IonatanSala**
Add breadcrumbs to marketplace using https://github.com/weppos/breadcrumbs_on_rails

### CODE REVIEW

File: app/controllers/shops_controller.rb [hunk 1]
```diff
@@ -2,6 +2,10 @@ class ShopsController < ApplicationController
   before_action :set_user_shop, only: [:edit, :update, :destroy]
   before_action :authenticate_user!, except: [:index, :show]
 
+  add_breadcrumb "Shops", :shops_path
+
+  add_breadcrumb "New Shop", :new_shop_path
```
Comment (danielpaul) [inline app/controllers/shops_controller.rb]: New shop wouldn't always be shown? Like when editing a shop and on single view of shop. This should be in the `new` method?

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -3,6 +3,10 @@ class ProductsController < ApplicationController
   before_action :set_shop
   before_action :set_product, only: [:edit, :update, :destroy]
 
+  add_breadcrumb "Products", :shop_products_path
+
+  add_breadcrumb "New Product", :new_shop_product_path
```
Comment (danielpaul) [inline app/controllers/products_controller.rb]: This should just be on the `new` method?

File: app/controllers/withdrawals_controller.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 class WithdrawalsController < ApplicationController
   before_action :set_shop, except: [:index]
 
+  add_breadcrumb "Withdrawals", :withdrawals_path
```
Comment (danielpaul) [inline app/controllers/withdrawals_controller.rb]: Could we have dashboard > withdrawls

File: app/controllers/shops_controller.rb (hunk 1 above)
Comment (IonatanSala) [inline app/controllers/shops_controller.rb]: What do you mean? To not show the "New Shop"?

Comment (danielpaul) [inline app/controllers/shops_controller.rb]: Go to a single shop page and you should be getting a similar breadcrumb: Shops > New Shop > My Cool Shop.

File: app/controllers/products_controller.rb (hunk 2 above)
Comment (IonatanSala) [inline app/controllers/products_controller.rb]: It is on the new method. The breadcrumb method looks at the path and determines the action from that.

File: app/controllers/shops_controller.rb (hunk 1 above)
Comment (IonatanSala) [inline app/controllers/shops_controller.rb]: It is getting that atm

Comment (danielpaul) [inline app/controllers/shops_controller.rb]: Yes, but it shouldn't be getting that tho. This breadcrumb for single shop page does not make sense: `Shops > New Shop > My Cool Shop`; whereas it should be this: `Shops > My Cool Shop`

**Pull request merged**